CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to contribute?
+
+## Code
+
+There is currently no styleguide for writing code but it would be great if we could reduce the number of issues as far
+as reasonable and then start to add more features.
+
+## Docs
+
+We are using Sphinx and are publishing the docs to [Readthedocs](https://cobbler.readthedocs.io). The source can be
+found in the `docs` folder.
+
+We want to try to use the following rules:
+
+- For first level headings should use `*` over and under the text.
+- For second level headings should use `#` under the text.
+
+## Website
+
+The website should contain promotional material and more complex guides. More information about how to contribute there
+you can find in the repo for the website: [Github](https://github.com/cobbler/cobbler.github.io)
+
+## Issues
```

Comment:
I think we should also ask for the version. The difference between 3.x and 2.x is pretty big.

Reply:
Addressed in cacc47e

Reply:
Sorry, that was a misunderstanding. I wasn't asking you to remove Fedora and Centos completely! :laughing: